backend/src/test/java/gov/cdc/usds/simplereport/api/queue/QueueMutationResolverTest.java

```diff
@@ -0,0 +1,95 @@
+package gov.cdc.usds.simplereport.api.queue;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.db.model.DeviceSpecimenType;
+import gov.cdc.usds.simplereport.db.model.DeviceType;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.auxiliary.DiseaseResult;
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+import gov.cdc.usds.simplereport.service.BaseServiceTest;
+import gov.cdc.usds.simplereport.service.DeviceTypeService;
+import gov.cdc.usds.simplereport.service.PersonService;
+import gov.cdc.usds.simplereport.service.TestOrderService;
+import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration;
+import gov.cdc.usds.simplereport.test_util.TestDataFactory;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@SliceTestConfiguration.WithSimpleReportStandardUser
+class QueueMutationResolverTest extends BaseServiceTest<PersonService> {
```

Comment:
Can anyone explain why I need to do this part? `extends BaseServiceTest<PersonService>` I copied it from the `OrganizationMutationResolverTest` file 😅 Otherwise my tests would blow up... 🧐

Reply:
The `BaseServiceTest` has a bunch of preconfigured utilities - it initializes some sample data in the database, defines the test as a SpringBoot test with specific properties, and sets up some commonly-used services and utilities like Okta or TestDataFactory. By extending that class, you get access to all those utilities. The SpringBoot test setup in particular is probably what caused this test to blow up if it wasn't included.
You may want to change the specific type to `BaseServiceTest<TestOrderService`, since we're not trying to test `PersonService`. (We're also not trying to test `TestOrderService`, but it is more likely to be used in the QueueMutation than `PersonService`...overall these mutation tests should probably get their own `Base` superclass, but that's a problem for another day)

Reply:
Ah got it! Thank you -- that is very helpful!!